packages/styles/panel.css

```
@@ -0,0 +1,58 @@
+:root {
+  --panel-border-color: var(--gray-40);
+  --panel-header-background-color: var(--gray-20);
+  --panel-header-color: var(--gray-90);
+  --panel-content-background-color: var(--white);
+  --panel-content-color: var(--gray-60);
+}
+
+.Panel {
+  border: 1px solid var(--panel-border-color);
+}
+
+.Panel__Header {
+  background-color: var(--panel-header-background-color);
+  color: var(--panel-header-color);
+  font-size: var(--text-size-normal);
+  display: flex;
+  height: 35px;
+  border-bottom: 1px solid var(--panel-border-color);
+}
+
+.Panel__Header-title {
+  flex-grow: 1;
+  display: flex;
+  line-height: var(--space-gutter);
+  margin-left: var(--space-small);
+}
```

```suggestion
.Panel__Header-title {
  flex-grow: 1;
  display: flex;
  padding: var(--space-half) 0;
}
```

The margin isn't needed because we're using flex/padding to space things appropriately. Without the line height/height changes from the container the `.Panel__Header` can "grow" appropriately if users make font size adjustments.

We can also add a min-height `min-height: var(--text-size-large-medium);` to enforce the minimum size of the header if we want to address that.